tests/timesignature_tests.ts

```diff
@@ -73,6 +75,27 @@ function big(options: TestOptions, contextBuilder: ContextBuilder): void {
   ok(true, 'all pass');
 }
 
+function additive(options: TestOptions, contextBuilder: ContextBuilder): void {
+  const ctx = contextBuilder(options.elementId, 400, 120);
+
+  new Stave(10, 10, 300).addTimeSignature('2+2+3/8').addTimeSignature('2+2+3/4+4').setContext(ctx).draw();
+
+  ok(true, 'all pass');
+}
+
+function complex(options: TestOptions, contextBuilder: ContextBuilder): void {
+  const ctx = contextBuilder(options.elementId, 400, 120);
+
+  new Stave(10, 10, 300)
+    .addTimeSignature('(2+3)/16')
+    .addTimeSignature('+')
+    .addTimeSignature('3/8')
```

Comment:
Can we add a few more different combinations here?

Reply:
Of course but I would like them to make sense. This is why I asked @mscuthbert. I will try to find some examples in internet.

Reply:
Gotcha. Is `+` supposed to be valid?

Reply:
yes, it is supposed to be valid. If we do not structure the complex signatures like that, parsing them would be really complex. The example above is one signature that corresponds to the complex one that @mscuthbert provided in #987.

Reply:
Examples added.